src/Sulu/Component/Content/Repository/ContentRepository.php

```diff
@@ -458,17 +458,22 @@ private function resolveResultPermissions(array $result, UserInterface $user = n
 
         foreach ($result as $index => $row) {
             $permissions[$index] = [];
-            $securityProperties = $row->getNode()->getProperties(SecuritySubscriber::SECURITY_PROPERTY_PREFIX . '*');
```

Comment:
for completion - this `$row->getNode()` caused the bad performance described in https://github.com/sulu/sulu/issues/5935, right? 🙂

Reply:
yes thats the bad performance!